packages/block-library/src/table/utils.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Normalize the row or column span value.
+ *
+ * @param {number|undefined} rowColSpan normalized value.
+ */
+export function normalizeRowColSpan( rowColSpan ) {
```

Comment:
What normalization is this doing? Should we have some unit tests for this function?

Reply:
I have clarified the purpose of this function and added unit tests in 29f244c.